arquillian-environment-reporter/arquillian-environment-reporter-impl/src/main/resources/org.arquillian.environment.reporter.impl.ArquillianEnvironmentKeys.properties

```diff
@@ -0,0 +1,7 @@
+ENVIRONMENT_SECTION_NAME.value=Environment
+JAVA_VERSION.value=Java Version
+TEST_RUNNER.value="Test Runner"
+DOCKER.value="Inside Docker"
+TIMEZONE.value="TimeZone"
+CHARSET.value="Default Charset"
+OPERATIVE_SYSTEM.value="Operative System"
```

Comment:
Don't add the quotation marks there (if not necessary) - the resulting JSON then contains `"\"Default Charset\""`